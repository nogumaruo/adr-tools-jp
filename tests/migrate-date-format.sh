adr init
(
	ADR_DATE=12/01/1992
	adr new With Old Date Format
)
adr new With Current Date Format
grep 日付: doc/adr/*
adr upgrade-repository
grep 日付: doc/adr/*
