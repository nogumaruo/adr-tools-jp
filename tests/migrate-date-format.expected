adr init
doc/adr/001-record-architecture-decisions.md
(
	ADR_DATE=12/01/1992
	adr new With Old Date Format
)
doc/adr/002-with-old-date-format.md
adr new With Current Date Format
doc/adr/003-with-current-date-format.md
grep 日付: doc/adr/*
doc/adr/001-record-architecture-decisions.md:日付: 1992-01-12
doc/adr/002-with-old-date-format.md:日付: 12/01/1992
doc/adr/003-with-current-date-format.md:日付: 1992-01-12
adr upgrade-repository
grep 日付: doc/adr/*
doc/adr/001-record-architecture-decisions.md:日付: 1992-01-12
doc/adr/002-with-old-date-format.md:日付: 1992-01-12
doc/adr/003-with-current-date-format.md:日付: 1992-01-12
